repos/system_upgrade/el8toel9/actors/targetuserspacecryptopolicies/libraries/targetuserspacecryptopolicies.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.common import mounting
+from leapp.libraries.stdlib import api, CalledProcessError
+from leapp.models import TargetUserSpaceInfo, CryptoPolicyInfo
+
+
+def _set_crypto_policy(context, current_policy):
+    if not os.path.exists(context.full_path('/usr/bin/update-crypto-policies')):
```

Comment:
Could you make this a constant?

Reply:
I am not fan of creation of constants that used just once in the whole code. but possibly we could use it for unit-tests.